src/adapters/serverbid.js

```diff
@@ -0,0 +1,166 @@
+import Adapter from 'src/adapters/adapter';
+import bidfactory from 'src/bidfactory';
+import bidmanager from 'src/bidmanager';
+import * as utils from 'src/utils';
+import { ajax } from 'src/ajax';
+
+var ServerBidAdapter = function ServerBidAdapter() {
+
+  var baseAdapter = Adapter.createNew('serverbid');
+
+  var BASE_URI = '//e.serverbid.com/api/v2';
+
+  var sizeMap = [null,
+    "120x90",
+    "120x90",
+    "468x60",
+    "728x90",
+    "300x250",
+    "160x600",
+    "120x600",
+    "300x100",
+    "180x150",
+    "336x280",
+    "240x400",
+    "234x60",
+    "88x31",
+    "120x60",
+    "120x240",
+    "125x125",
+    "220x250",
+    "250x250",
+    "250x90",
+    "0x0",
+    "200x90",
+    "300x50",
+    "320x50",
+    "320x480",
+    "185x185",
+    "620x45",
+    "300x125",
+    "800x250"
+  ];
+
+  var bidIds = [];
+
+  baseAdapter.callBids = function(params) {
+
+    if (params && params.bids && utils.isArray(params.bids) && params.bids.length) {
+
+      var data = {
+        placements: [],
+        time: Date.now(),
+        user: {},
+        url: document.location.href,
```

Comment:
I would recommend using these function utils.getTopWindowUrl() and utils.getTopWindowLocation()